src/pipelines/_common/wikidata.py

```diff
@@ -30,7 +30,7 @@ def _process_item(props: Dict[str, str], key_wikidata: Tuple[str, str]):
         key, wikidata_id = key_wikidata
         return {"key": key, **wikidata_properties(props, wikidata_id)}
 
-    def parse(self, sources: List[str], aux: Dict[str, DataFrame], **parse_opts):
+    def parse(self, sources: List[str], aux: Dict[str, DataFrame], **parse_opts) -> DataFrame:
         data = aux["knowledge_graph"].merge(aux["metadata"])[["key", "wikidata"]].set_index("key")
```

Comment:
Might be worth splitting this into multiple lines by function call for readability.

Reply:
I'm using `black` so I'm trying not to deal with formatting issues. Ideally, we would tweak the file which configures the formatter for anything like this: https://github.com/open-covid-19/data/blob/master/src/.pre-commit-config.yaml

It is currently set to max 100 chars per line because I found 80 to be a bit oppressing. But I understand you are looking to meet readability requirements so I'm happy to compromise and do formatting your way.